examples/troubleshooting/README.md

```diff
@@ -16,26 +16,35 @@ dependencies will allow for easy compilation
 
 (Manual compilation would require the jars and their dependencies downloaded, and then javac -cp <list of jars> path-to-java-file.)
 
-# Traces
+# Traces, Metrics, Logs
 
-The [`TestOtelSdkTrace`](./src/main/java/elastic/troubleshooting/TestOtelSdkTrace.java) class is a standalone class that creates
-a span (named `test span`) in a service that you name. It takes
-three arguments
+The examples in this section all take the same three arguments:
 1. The name of the service you want to have displayed in the APM UI
 2. The endpoint to send traces to, normally the Elastic APM server or the OpenTelemetry collector.
-   The url would typically look like `http://localhost:4318/v1/traces` or `https://somewhere:443/v1/traces` 
-   but if `/v1/traces` is missing from the argument, it is added to the endpoint 
-3. The secret token or apikey in the format secret:<token> or apikey:<apikey>. 
-   If the argument doesn't start with neither `secret:` nor `apikey:`, then the full argument is assumed to be a secret token
+   The url would typically look like `http://localhost:4318/v1/traces` or `https://somewhere:443/v1/traces`
+   but if `/v1/traces` is missing from the argument, it is added to the endpoint
```

Comment:
do we really need to document the full path here ? also if we are sending more than one signal having `/traces` in it is a bit confusing.